rules/prevent-abbreviations.js

```diff
@@ -116,7 +116,7 @@ const getNameReplacements = (name, options, limit = 3) => {
 	}
 
 	// Split words
-	const words = name.split(/(?=[^a-z])|(?<=[^A-Za-z])/).filter(Boolean);
+	const words = name.split(/(?=\P{Ll})|(?<=\P{L})/u).filter(Boolean);
```

Comment:
Use the verbose names (`Lowercase_Letter`), for readability.

Reply:
Updated to include the verbose names

Reply:
We should make a rule for it 😄

Reply:
https://github.com/ota-meshi/eslint-plugin-regexp/issues/720

Reply:
I'll work on it actually

Reply:
Note: the proposal is on another repo.